src/common/engine.ts

```diff
@@ -75,7 +75,7 @@ export class CurrencyEngine<T> {
   log: EdgeLog
   warn: (message: string, e?: Error) => void
   error: (message: string, e?: Error) => void
-  otherData: Object
+  otherData: { [key: string]: any }
```

Comment:
We might want to cherry-pick this first commit as its own PR. Seems like a pretty juicy cleanup that shouldn't depend on Tron getting in.